src/BizHawk.Client.EmuHawk/MainForm.cs

```diff
@@ -619,9 +613,17 @@ _argParser.SocketAddress is var (socketIP, socketPort)
 				Config.MainWndy = 0;
 			}
 
-			if (Config.MainWndx != -1 && Config.MainWndy != -1 && Config.SaveWindowPosition)
+			if (Config.SaveWindowPosition)
 			{
-				Location = new Point(Config.MainWndx, Config.MainWndy);
+				if (Config.MainWndx is int x && Config.MainWndy is int y)

```

Comment:
Was it necessary to make these nullable to implement this new feature? If you could continue using `-1` as the sentinel value instead of `null`, I think that would be preferable. Otherwise, try `Point?` instead of 2 `int?`s.

Reply:
It's not necessary to the PR, no. I just made this change because negative values including -1 are valid window positions

Reply:
Possibly related to https://github.com/TASEmulators/BizHawk/issues/3544?

Reply:
Not directly, tool window position saving is handled elsewhere